
CommandNotFoundError: Your shell has not been properly configured to use 'conda activate'.
To initialize your shell, run

    $ conda init <SHELL_NAME>

Currently supported shells are:
  - bash
  - fish
  - tcsh
  - xonsh
  - zsh
  - powershell

See 'conda init --help' for more information and options.

IMPORTANT: You may need to close and restart your shell after running 'conda init'.


/nfs/home/mrobo/anaconda3/envs/omm100/lib/python3.9/site-packages/simtk/openmm/app/internal/pdbstructure.py:443: UserWarning: WARNING: two consecutive residues with same number (ATOM   4698  OH2 TIP3    1      11.878  10.016   6.880  1.00  0.00      SOLV    , ATOM   4697 H126 LIG     1      52.368  32.492  31.940  1.00  0.00      LIG     )
  warnings.warn("WARNING: two consecutive residues with same number (%s, %s)" % (atom, self._current_residue.atoms[-1]))
####### SITE NAMES #######
[[], ['C029', 'H030', 'C031', 'C032', 'H033', 'C034', 'S035', 'C036', 'H037', 'C038', 'C039', 'H040', 'H041', 'H042', 'N043'], ['C044', 'H045', 'C046', 'C047', 'H048', 'N049', 'H050', 'C051', 'O052', 'O053', 'C054', 'H055', 'H056', 'H057'], ['C058', 'H059', 'C060', 'C061', 'H062', 'N063', 'H064', 'C065', 'O066', 'O067', 'C068', 'H069', 'H070', 'C071', 'H072', 'H073', 'H074'], ['C075', 'H076', 'C077', 'C078', 'H079', 'C080', 'N081', 'C082', 'H083', 'C084', 'C085', 'H086', 'C087', 'H088', 'C089', 'H090', 'H091', 'H092'], ['C093', 'H094', 'C095', 'C096', 'H097', 'C098', 'N099', 'N100', 'C101', 'C102', 'H103', 'C104', 'H105', 'C106', 'H107', 'H108', 'H109'], ['C110', 'H111', 'C112', 'C113', 'H114', 'C115', 'N116', 'C117', 'H118', 'C119', 'C120', 'H121', 'N122', 'C123', 'H124', 'H125', 'H126']]
sub2site
[0, 1, 1, 1, 1, 1, 1]
Traceback (most recent call last):
  File "/nfs/home/mrobo/negs_cleanup/3.gen_omm_patch/setupSystem_NBFIXs.py", line 455, in <module>
    setup_NBFIXs(83, 'prep/', trunc=True)
  File "/nfs/home/mrobo/negs_cleanup/3.gen_omm_patch/setupSystem_NBFIXs.py", line 251, in setup_NBFIXs
    nonbond[0].addTabulatedFunction('lj12a',Discrete2DFunction(len(type_list),len(type_list),lj12a))
NameError: name 'Discrete2DFunction' is not defined

real	19m37.116s
user	19m36.053s
sys	0m0.440s
FINISHED
